lib/StateStore.js

```diff
@@ -142,23 +142,27 @@ class StateStore {
     return this._delete(key)
   }
 
-  // /**
```

Comment:
should you delete the commented out CosmosStateStore.keys() code as well? https://github.com/adobe/aio-lib-state/blob/8aade8ba742436896ed15cc0c0f1fc68610cc228/lib/impl/CosmosStateStore.js#L152

Reply:
sure thanks! removed